sklearn/cross_decomposition/pls_.py

```diff
@@ -431,8 +438,9 @@ def predict(self, X, copy=True):
         # Normalize
         X -= self.x_mean_
         X /= self.x_std_
-        Ypred = np.dot(X, self.coef_)
-        return Ypred + self.y_mean_
+        Ypred = safe_sparse_dot(
```

Comment:
Is the safe_sparse_dot needed?

Reply:
To be honest, it was mostly because I changed the coef_ attribute to match the behavior of LinearRegression.coef_. And LinearRegression is using safe_sparse_dot. But I guess it was not necessary. Do you want me to revert this change ?